src/applications/vaos/new-appointment/components/VAFacilityPage/EligibilityModal.jsx

```diff
@@ -30,31 +59,28 @@ export default function EligibilityModal({
       <div aria-atomic="true" aria-live="assertive">
         You’ll need to call your VA health facility to schedule this
         appointment. Not all VA facilities offer online scheduling for all types
-        of care
+        of care.
       </div>
     );
   } else if (requestReason === ELIGIBILITY_REASONS.noRecentVisit) {
+    const monthRequirement =
+      (facilityDetails.legacyVAR.settings[typeOfCare.id].request
+        .patientHistoryDuration /
+        365) *
+      12;
     title = 'We can’t find a recent appointment for you';
     content = (
-      <>
-        <p>
-          You need to have visited this facility within the past{' '}
-          {monthRequirement} months for {lowerCase(typeOfCare?.name)} to request
-          an appointment for this type of care.
-        </p>
-        <p>
-          You’ll need to call the facility to schedule this appointment. Or{' '}
-          <NewTabAnchor href="/find-locations">
-            search for another VA facility
-          </NewTabAnchor>
-          .
-        </p>
-      </>
+      <div aria-atomic="true" aria-live="assertive">
```

Comment:
Making these messages consistently use aria-live and updating the recent visit text here to match the DS one.